src/Akka.HealthCheck/AkkaHealthCheck.cs

```diff
@@ -102,16 +102,19 @@ public AkkaHealthCheck(HealthCheckSettings settings, ExtendedActorSystem system)
         public static IActorRef StartTransportActor(ITransportSettings settings, ExtendedActorSystem system,
             ProbeKind probeKind, IActorRef probe)
         {
+            var  _log = system.Log;
```

Comment:
No need to use a local variable here. Just call `ActorSystem.Log`.

If you can create a custom `ILoggingAdapter` that names itself "AkkaHealthCheck` that would be ideal and stick it into a private readonly field that would be ideal - that way it's clear where the log source comes from.

I would also make the logging level configurable - turning on `akka.loglevel = DEBUG` turns on a massive firehose of log messages coming from all sorts of systems and this data will get lost in the shuffle. Better to have a HOCON option that specifies the loglevel to be used for all debug / diagnositic logging used by Akka.HealthCheck's internals (i.e. everything that runs inside the built-in transports, built-in probes, AND the `AkkaHealthCheck` plugin itself).

By default this value should be `Debug`, but a user can change the HOCON setting to change it to `INFO` if need be. I think that'd be a better option - keeps the verbosity under control.